templates/frontend/objects/article_details.tpl

```diff
@@ -240,7 +240,20 @@
 											</a>
 										</li>
 									{/foreach}
-							  </ul>
+									 {* Download citation formats *}
+									{if count($citationDownloads)}
+										<div class="panel-heading">{translate key="submission.howToCite.downloadCitation"}</div>
```

Comment:
I don't think we can put a `<div>` directly beneath a `<ul>` like this. We either need to use two `<ul>` lists or drop the heading. The icon should be sufficient to visually distinguish the download links.

Reply:
Regarding the icons, I noticed that bootstrap3 doesn't import the fontawesome lib properly so I added some extra changed in BootstrapThreeThemePlugin.inc.php file.
Looking forward to your feedback.